ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -1497,6 +1621,43 @@
 			};
 			name = Release;
 		};
+		3A3CB9121EF76B01008DAC6D /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CODE_SIGN_ENTITLEMENTS = AllAboutOlafWidget.entitlements;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEVELOPMENT_TEAM = 363G7T9TGX;
+				INFOPLIST_FILE = "AllAboutOlafWidget/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
```

Comment:
Hard to say. Cursory glance at `master` .pbxproj shows 8.1 referenced three times and 10.1 referenced two times. I'll look closer later.